9.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-unused-vars */
+
+// Q9: There exists exactly one Pythagorean triplet for which a + b + c = 1000. Find the product of abc. 
+
+// function to check if pythagoream triple , received array, returns boolean 
+
+function isPyTriple(arr){
+    let a = Math.pow(arr[0],2)
```

Comment:
To prevent programming errors, it's common in functions like this to throw an exception of the arr is not length 3.